pom.xml

```diff
@@ -119,7 +119,7 @@
     <mockito.version>1.9.5</mockito.version>
     <mysql.version>5.1.21</mysql.version>
     <netty.version>3.6.6.Final</netty.version>
-    <netty.http.version>0.10.0</netty.http.version>
+    <netty.http.version>0.11.0-SNAPSHOT</netty.http.version>
```

Comment:
we should not have snapshot dependencies. Instead we should release anew version of netty-http.